lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -302,6 +302,14 @@ RUN cd json && \
     cd ../ && rm -Rf _build && \
     ldconfig --verbose
 
+# Install bazel
+WORKDIR /usr/sbin
+RUN wget --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-amd64 && \
+    chmod +x bazelisk-linux-amd64 && \
+    ln -sf /usr/sbin/bazelisk-linux-amd64 /usr/sbin/bazel
+
+RUN mkdir -p /usr/lib/x86_64-linux-gnu/ && ln -s /usr/lib/libglog.so /usr/lib/x86_64-linux-gnu/libglog.so.0
```

Comment:
@vktng 

I've added `ln -s /lib64/libgnutls.so /usr/lib/libgnutls.so` here to make it work.